dip-0003.md

```diff
@@ -77,6 +77,25 @@ The ProRegTx contains 3 public key IDs, which represent 3 different roles in the
 
 A single key can be used for multiple roles so the owner can also operate the masternode and/or issue proposal votes. Alternately, different keys can be used to delegate duties to other people. For example, owners might decide to use a masternode hosting service instead of self-hosting the masternode. To do so, they would use the operator key provided by the hosting service. The same applies to the voting key, which allows them to assign voting rights to someone else.
 
+The collateralOutpoint of the ProRegTx might refer to an external collateral or to an output which is part of the ProRegTx itself. If the hash of collateralOutpoint is null, it refers to
+an output of the ProRegTx. If it refers to an external collateral (hash of collateralOutpoint is not null), the collateral must exist and not be spent. The ProRegTx must also prove ownership
+of external collaterals by setting payloadSigSize and payloadSig to a valid signature which signs the following message:
+
+`<magicString><payoutStr>|<operatorReward>|<ownerKeyAddress>|<votingKeyAddress>|<payloadHash>`
+
+The individual parts of the message are:
+1. `<magicString>`: A fixed string which is always "DarkCoin Signed Message:" with a newline character appended.
```

Comment:
Maybe clarify here that magic string is added just like for any other text message signed via standard interface (i.e. `signmessage` rpc in the reference client or `SignMessage` method in HW wallets), so it should only be added if you are building the signature from the ground up and it should NOT be added if you are using standard message signing interface. Or smth like that, you get the idea I guess :)

Reply:
Added a commit on top that adds a note about this.